docs/adding_an_env_var.md

```diff
@@ -0,0 +1,14 @@
+## Adding an ENV var
+
+### Server side only
+
+1. Add the variable to your `.env` file.
+1. Edit `src/desktop/config.coffee` to include default for your env variable. You can safely use `null` to indicate an empty variable.
+1. Use it in the runtime via `/src/config.js`
+
+### Server + Client
+
+1. Add the variable to your `.env` file.
+1. Edit `src/desktop/config.coffee` to include default for your env variable. You can safely use `null` to indicate an empty variable.
+1. Edit `src/lib/setup_sharify.js` to export your env variable to the client runtime
+1. You can access it via `sd.[YOUR_VAR]` in client code
```

Comment:
All list items are ichi-ban! 😉

Reply:
They get transformed by markdown, so if you always write `1.` then you can re-order any time with no pain

1. "1."
1. "1."
1. "1."

Reply:
Oooohhh, TIL!

Reply:
Nice, thank you! I always forget/have to remember the 3ish places to update when adding an ENV var.